var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -24,6 +24,7 @@
 ##############################################################################
 from spack import *
 from spack import architecture
+from spack.build_environment import dso_suffix
```

Comment:
Is this not already available without the import? No other package that use `dso_suffix` imports it.

Reply:
Yep you are right, I just copied from the first `package.py` without realising that it was the one in `lib` :grml: